yal/scripts/update_rapidpro_contacts_persona_emoji.py

```diff
@@ -0,0 +1,63 @@
+import argparse
+import json
+
+import emoji
+import requests
+
+from yal.utils import extract_first_emoji
+
+
+def get_arguments() -> argparse.Namespace:
+    parser = argparse.ArgumentParser(
+        description="Pretends to be a webhook from whatsapp for a template message "
+        "button click with payload"
```

Comment:
Should probably change this description to something that matches what this does.